augly/video/helpers/ffmpeg.py

```diff
@@ -5,6 +5,7 @@
 import math
 import os
 import shutil
+import numpy as np
```

Comment:
i think this was left here by accident while you were resolving merge conflicts with the `main` branch
```suggestion
```